tests/test_pretrial_verified_law_rescue.py

```
@@ -0,0 +1,229 @@
+
+def test_verified_basis_projection_removes_internal_research_metadata() -> None:
+    basis = verified_legal_basis_from_research(_verified_research())
+    assert basis == ["Обязательство должно исполняться надлежащим образом (статья 272 ГК РК)."]
+    assert "текст нормы" not in basis[0]
+    assert "источник:" not in basis[0]
+
+
+def test_verified_basis_projection_localizes_kazakh_article_label() -> None:
+    claim = verified_claim_line(
+        "Міндеттеме тиісті түрде орындалуға тиіс.",
+        "статья 272 ГК РК",
```

Ruff (0.16.2)

[warning] String contains ambiguous `Р` (CYRILLIC CAPITAL LETTER ER). Did you mean `P` (LATIN CAPITAL LETTER P)?

(RUF001)